policyengine_us/parameters/gov/irs/deductions/deductions_if_itemizing.yaml

```diff
@@ -8,6 +8,7 @@ values:
     - casualty_loss_deduction
     - qualified_business_income_deduction
     - wagering_losses_deduction
+    - tuition_and_fees_deduction
```

Comment:
make a 2021-01-01: 
  2021-01-01:
    - charitable_deduction
    - interest_deduction
    - salt_deduction
    - medical_expense_deduction
    - casualty_loss_deduction
    - qualified_business_income_deduction
    - wagering_losses_deduction

Reply:
same for not itemized to show that it was repealed